acsf-convert/files/factory-hooks/post-install/post-install.php

```diff
@@ -63,13 +64,14 @@ function error($message) {
 
 $docroot = sprintf('/var/www/html/%s.%s/docroot', $site, $env);
 
+// phpcs:ignore
 $cache_directory = exec("/usr/bin/env php /mnt/www/html/$site.$env/vendor/acquia/blt/scripts/blt/drush/cache.php $site $env $uri");
-
+// phpcs:ignore
 shell_exec(sprintf('mkdir -p %s', escapeshellarg($cache_directory)));
```

Comment:
Yes it is OOB. I don't tell anyone about it since our files would be littered with ignores. One thing to note is that not everything Acquia produces and adds to our code base passes phpcs. In fact, a lot does not -- ask me how I know. :|

Reply:
^^ What he said.

Reply:
What you're both leaving out is that a lot of the Drupal codebase won't pass the Drupal coding conventions....